RecoHGCal/TICL/plugins/PatternRecognitionbyFastJet.cc

```diff
@@ -53,7 +58,7 @@ void PatternRecognitionbyFastJet<TILES>::buildJetAndTracksters(std::vector<Pseud
   }
 
   auto trackster_idx = result.size();
-  auto jetsSize = std::count_if(jets.begin(), jets.end(), [=](fastjet::PseudoJet jet) {
+  auto jetsSize = std::count_if(jets.begin(), jets.end(), [=, this](fastjet::PseudoJet jet) {
```

Comment:
Also here the warning suppression could be avoided
```suggestion
  auto jetsSize = std::count_if(jets.begin(), jets.end(), [this](fastjet::PseudoJet jet) {
```
or pass the `minNumLayerCluster_` by copy explicitly.